src/index.html

```diff
@@ -7,6 +7,12 @@
   <title>Baratheon</title>
 </head>
 <body>
-  <button class="btn-danger">hey</button>
+  <nav class="navbar sticky-top navbar-light bg-light">
+    <form class="d-flex flex-row justify-content-between col-12">
+      <img src="https://f4.bcbits.com/img/0009711024_10.jpg" width="50" height="50" class="flex-row align-top col-1" id="navImg">
+      <input class="form-control mr-sm-2 col-5" id="chatInput" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0 col-1" id="clearChat" type="submit">Clear Chat</button>
```

Comment:
we probably will want the take the `type='submit'` off of this button since it causes the page to refresh when you click it

Reply:
This has been updated.